android/app/src/main/java/org/pathcheck/covidsafepaths/bridge/EventSender.kt

```diff
@@ -12,7 +12,7 @@ object EventSender {
     fun sendExposureNotificationStatusChanged(reactContext: ReactContext?, enabled: Boolean) {
         reactContext
             ?.getJSModule(RCTDeviceEventEmitter::class.java)
-            ?.emit(EN_STATUS_CHANGED_EVENT, Util.getEnStatusWritableArray(enabled))
+            ?.emit(EN_STATUS_CHANGED_EVENT, enabled ? CallbackMessages.EN_STATUS_ACTIVE : CallbackMessages.EN_STATUS_DISABLED)
```

Comment:
@juancruzgs: @johnschoeman and I updated based on your comments. Can you please do one more quick review and confirm this is all set? Thank you!